#class Exception(object):
#    pass

class ValueError(Exception):
    pass

raise ValueError(), "hi"
